mempool/metrics.go

```diff
@@ -11,17 +11,6 @@ const (
 	// MetricsSubsystem is a subsystem shared by all metrics exposed by this
 	// package.
 	MetricsSubsystem = "mempool"
-
-	TypeLabel = "type"
-
-	FailedPrecheck = "precheck"
-	FailedAdding   = "adding"
-	FailedRecheck  = "recheck"
-
-	EvictedNewTxFullMempool      = "full-removed-incoming"
-	EvictedExistingTxFullMempool = "full-removed-existing"
-	EvictedTxExpiredBlocks       = "expired-ttl-blocks"
-	EvictedTxExpiredTime         = "expired-ttl-time"
```

Comment:
[question] is it technically breaking to remove these constants? I think it's fine to do if we're confident no downstream consumers use these constants.

It likely would've been easier/safer if we didn't make them public to begin with.